bigquery/snippets/snippet.go

```diff
@@ -472,6 +710,49 @@ func deleteTable(client *bigquery.Client, datasetID, tableID string) error {
 	return nil
 }
 
+func deleteAndUndeleteTable(client *bigquery.Client, datasetID, tableID string) error {
+	ctx := context.Background()
+	// [START bigquery_undelete_table]
```

Comment:
Nice. I presume this is slotted for https://cloud.google.com/bigquery/docs/managing-tables#undeletetable ?

Reply:
That's the plan.